Revert to old license syntax for support with older setuptools

author: micketeer@gmail.com
--- pyproject.toml.orig 2025-06-24 11:01:08.103528554 +0000
+++ pyproject.toml      2025-06-24 11:01:26.290952440 +0000
@@ -18,8 +18,7 @@
     "spglib",
     "phonopy>=2.39,<2.40",
 ]
-license = "BSD-3-Clause"
-license-files = ["LICENSE"]
+license = {file = "LICENSE"}
 
 [project.urls]
 Homepage = "https://phonopy.github.io/phono3py/"

